docs/datadoc/keywords.md

```diff
@@ -11,6 +11,17 @@ The meaning of the columns are as follows:
 - **Definition**: The definition of the keyword.
 - **Usage note**: Notes about how to use the keyword.
 
+## Special keywords (from JSON-LD)
+See the [JSON-LD specification] for more details.
+
+| Keyword    | Range         | Conformance | Definition                                                   | Usage note |
+|------------|---------------|-------------|--------------------------------------------------------------|------------|
+| [@id]      | IRI           | mandatory   | IRI identifying the resource to document.                    |            |
+| [@type]    | IRI           | recommended | Ontological class defining the class of a node.              |            |
+| [@context] | dict&#124list | optional    | Context defining namespace prefixes and additional keywords. |            |
+| [@base]    | list          | optional    | Base IRI against which relative IRIs are resolved.           |            |
```

Comment:
is this namespace?

Reply:
Yes, it is a kind of default namespace, similar to `base` in turtle. It will be applied to keywords with no pre-defined mappings in the context.

Reply:
JSON-LD also has a `@vocab` keyword that may be even more useful.